src/Components/Backoffice/Users/BackofficeUsers.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import {Table, Thead, Tbody, Tr, Th, Td} from 'react-super-responsive-table'
+import 'react-super-responsive-table/dist/SuperResponsiveTableStyle.css'
+import "../../../Styles/BackOfficerUsersTable.css";
+import Button from "@material-ui/core/Button/Button";
+import { Link } from 'react-router-dom';
+
+const BackofficeUsers = () => {
+    const createData =(nombre, id, email,) => {
+        return { nombre, id, email, };
+      }
+      const rows = [
+        createData('Nombre 0', 159, "nombre@alkemy.org",),
+        createData('Nombre 1', 237, "nombre@alkemy.org",),
+        createData('Nombre 2', 262, "nombre@alkemy.org",),
+      ];
```

Comment:
this should come from a mock array and iterate over that mock array to create the data so you emulate an endpoint response